lib/index.js

```diff
@@ -437,7 +437,7 @@ module.exports.renderSync = function(opts) {
     return result;
   }
 
-  throw util._extend(new Error(), JSON.parse(result.error));
+  throw assign(new Error(), JSON.parse(result.error));
```

Comment:
Yup, the idea was to have the error object thrown (or return in case of asynchornous call) to the caller. We tried to avoid JSON parsing on native side and casting it to object in JavaScript using `util._extend` (without having to know what is in the object, line/column/message).

`lodash.assign` approach is the best backward compatible solution, if we are already depending on that lib.